src/detect-engine-uint.c

```diff
@@ -137,9 +147,17 @@ DetectU32Data *DetectU32Parse (const char *u32str)
                     return NULL;
 
                 if (arg2[0] == '<') {
-                    u32da.mode = DETECT_UINT_LT;
+                    if (arg2[1] == '=') {
```

Comment:
Thanks for these.
If you wish, could you remove the C implementation and use the Rust one from
https://github.com/OISF/suricata/blob/master/rust/src/http2/parser.rs#L757

Reply:
Hmm ok, a few questions:

Then i would call this rust-function from C in the *Setup()-function of the keyword?
This looks pretty generic, will this stay in the http2/parser?
Will detect-engine-uint.h/.c be deprecated in the future?

Reply:
First, do not feel forced to do this...

If you want it, then 

> i would call this rust-function from C in the *Setup()-function of the keyword?

yes, and from the match function as well I guess

> This looks pretty generic, will this stay in the http2/parser?

No, if you can move it to somewhere generic, I would appreciate it

> Will detect-engine-uint.h/.c be deprecated in the future?

Yes, replace it with the rust version

So, do as you wish

Reply:
hmmm i would stick to the implemented solution, sry feels a bit more mature :)

Currently I've much other stuff on the table hope to get the v10 done soon....